test/tests/SSLTest.cpp

```diff
@@ -63,8 +63,12 @@ struct SSLTest : tpunit::TestFixture {
         const string proxy = "127.0.0.1:3128";
 
         // Create a transaction with a socket, send the above request.
-        SStandaloneHTTPSManager::Transaction* transaction = new SStandaloneHTTPSManager::Transaction(manager);
-        transaction->s = new SHTTPSProxySocket(proxy, host);
+        SStandaloneHTTPSManager::Transaction* transaction = new SStandaloneHTTPSManager::Transaction(manager, "proxyTest");
```

Comment:
I don't think we automatically send transactions in _httpSend which is used in auth: 

https://github.com/Expensify/Bedrock/blob/7c6ed108eca4ac74015d605b8efff41df4d60437/libstuff/SHTTPSManager.cpp#L205

Reply:
NVM, https://github.com/Expensify/Bedrock/blob/745757cfccc4be8e753a1d70bafbe914d63b960a/libstuff/SHTTPSManager.cpp#L175-L176 we